src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -107,6 +107,21 @@ class TestRequestRateManager : public TestLoadManagerBase,
     }
   }
 
+  void TestCalculateThreadIds(std::vector<size_t>& expected_thread_ids)
+  {
+    std::vector<size_t> actual_thread_ids =
+        RequestRateManager::CalculateThreadIds();
+    if (on_sequence_model_) {
```

Comment:
No need for this logic here. Just compare to the passed in expected_thread_ids.size()

Reply:
updated